search-api/src/search_api/request_handlers/document_access_request_handler.py

```diff
@@ -70,18 +70,22 @@ def create_invoice(document_access_request: DocumentAccessRequest, user_jwt: Jwt
                                           header, business_json)
 
         if payment_response.status_code in (HTTPStatus.OK, HTTPStatus.CREATED):
-            payment_completion_date = datetime.utcnow()
+            is_pad = payment_response.json().get('paymentMethod') == 'PAD'
+            today_utc = datetime.utcnow()
```

Comment:
`datetime.now(tz=timezone.utc)`
datetime.utcnow() = depreciated

Reply:
Yeah, pycharm shows the same !
I wasn't sure about updating it in just one place, but you are right. When changing the code, update it in place I changed if it does not break everything else :D